packages/core/src/Breadcrumb.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Icon from 'pcln-icons'
+import Text from './Text'
+
+const Container = styled(Flex)`
+  cursor: ${props => (props.onClick ? 'pointer' : 'default')};
+`
+
+function Breadcrumb({ className, isCurrent, isCompleted, children, onClick }) {
+  const color = isCurrent || isCompleted ? 'primary' : 'text.light'
+
+  return (
+    <Container className={className} alignItems="center" onClick={onClick}>
+      {isCompleted && <CheckIcon />}
+      <Text color={color} bold={isCurrent} fontSize={1}>
+        {children}
+      </Text>
+    </Container>
+  )
+}
+
+const CheckIcon = () => (
+  <Icon name="check" color="primary" size={16} mr={1} legacy={false} />
```

Comment:
Since pcln-icons supports named icon, I think this could become CheckIcon.

Reply:
I can make this work if I import from `pcln-icon/lib/Check` but not from just `pcln-icons` as it's not exported to handle this.

Reply:
Cool, would you mind doing it in this PR? If you prefer to keep the Pr the same scope, would you mind opening a new issue to track for next volunteer to follow up to use names icon import?

Reply:
Sorry I'm not sure if you want me to change the import to `pcln-icon/lib/Check` or to update the icons package to export the icons differently.